downstream issue when linking against rocsolver

See: https://github.com/rocm-arch/hipmagma/issues/3

Snippet of error logs:
```
[1547/3370] Linking CXX executable testing/testing_slacpy
FAILED: testing/testing_slacpy 
: && /usr/bin/hipcc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -fcf-protection=none -std=c++11 -fopenmp=libomp -Wall -Wno-unused-function -O3 -DNDEBUG -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now    -rdynamic CMakeFiles/testing_slacpy.dir/testing/testing_slacpy.cpp.o -o testing/testing_slacpy  -Wl,-rpath,/build/hipmagma/src/magma-2.6.2/build/lib:/opt/rocm/hipblas/lib:/opt/rocm/hipsparse/lib  lib/libtester.so  lib/liblapacktest.so  lib/libmagma.so  /opt/rocm/hip/lib/libamdhip64.so.5.1.20531  -Wl,-Bstatic  -lclang_rt.builtins-x86_64  -Wl,-Bdynamic  /opt/rocm/hipblas/lib/libhipblas.so.0.1  /opt/rocm/hipsparse/lib/libhipsparse.so.0.1  -llapack  -lblas && :
/usr/bin/ld: warning: libfmt.so.8, needed by /opt/rocm/lib/librocsolver.so.0, not found (try using -rpath or -rpath-link)
/usr/bin/ld: /opt/rocm/lib/librocsolver.so.0: undefined reference to `rocblas_status_ rocblas_internal_gemm_template<false, double, double, double>(_rocblas_handle*, rocblas_operation_, rocblas_operation_, int, int, int, double const*, double const*, long, int, long, double const*, long, int, long, double const*, double*, long, int, long, int)'
```

I wonder if `options=(!lto)` need to be added to rocsolver and/or rocblas.

--- a/rocblas/.SRCINFO
+++ b/rocblas/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = rocblas
 	pkgdesc = Next generation BLAS implementation for ROCm platform
 	pkgver = 5.1.1
-	pkgrel = 1
+	pkgrel = 2
 	url = https://rocblas.readthedocs.io/en/latest
 	arch = x86_64
 	license = MIT
@@ -16,6 +16,7 @@ pkgbase = rocblas
 	makedepends = gcc-fortran
 	depends = hip
 	depends = openmp
+	options = !lto
 	source = rocblas-5.1.1.tar.gz::https://github.com/ROCmSoftwarePlatform/rocBLAS/archive/rocm-5.1.1.tar.gz
 	sha256sums = 8a59c6f1ca36ccaf793bada022641a26da568249c776bd319d0d5bac6a8a809a
 

--- a/rocblas/PKGBUILD
+++ b/rocblas/PKGBUILD
@@ -2,7 +2,7 @@
 # Contributor: Markus Näther <naether.markus@gmail.com>
 pkgname=rocblas
 pkgver=5.1.1
-pkgrel=1
+pkgrel=2
 pkgdesc='Next generation BLAS implementation for ROCm platform'
 arch=('x86_64')
 url='https://rocblas.readthedocs.io/en/latest'
@@ -13,6 +13,7 @@ makedepends=('cmake' 'git' 'python' 'python-virtualenv' 'python-pyaml'
 _rocblas='https://github.com/ROCmSoftwarePlatform/rocBLAS'
 source=("$pkgname-$pkgver.tar.gz::$_rocblas/archive/rocm-$pkgver.tar.gz")
 sha256sums=('8a59c6f1ca36ccaf793bada022641a26da568249c776bd319d0d5bac6a8a809a')
+options=(!lto)
 _dirname="$(basename "$_rocblas")-$(basename "${source[0]}" ".tar.gz")"
 
 build() {

